packages/lesswrong/lib/vulcan-users/permissions.ts

```diff
@@ -169,6 +169,13 @@ export const userOverNKarmaFunc = (n: number) => {
     }
 }
 
+export const userOverNKarmaOrApproved = (n: number) => {
```

Comment:
Can this re-use `userOverNKarmaFunc` and just add the additional `reviewedByUserId` condition?